static/app/views/starfish/views/webServiceView/spanGroupBreakdownContainer.tsx

```diff
@@ -88,7 +89,7 @@ export function SpanGroupBreakdownContainer({transaction, transactionMethod}: Pr
     ),
     orgSlug: organization.slug,
     referrer: 'api.starfish-web-service.total-time',
-    location,
+    location: omit(location, 'query.cursor'),
```

Comment:
the span breakdown chart still reloads on pagination, we can probably avoid that by omitting the cursor in the `useEventsStatsQuery` below too!

Reply:
🤔 useEventsStatsQuery doesn't take `location` but calls `getLocation` inside of itself.

The new version of this page doesn't use this graph anyways so I'll just skip this for now, but if I bring it back I'll update the chart as well